samples/bluetooth/mesh/light/README.rst

```diff
@@ -144,6 +147,22 @@ To review the required configuration alterations, open and inspect the two files
 For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 For more information about selecting a sysbuild configuration file, see the Sysbuild Kconfig file section on the :ref:`zephyr:sysbuild` page in the Zephyr documentation.
 
+nRF53 Development Kit
+*********************
+
+To enable the DFU feature for the nRF53 development kit, set :makevar:`SB_CONF_FILE` to :file:`sysbuild-dfu.conf` and set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
+Additionally, you need to set the :makevar:`EXTRA_CONF_FILE` for the ipc_radio network image to :file:`overlay-dfu.conf` as well.
+This is an additional network image specific configuration overlay file which allocates the necessary resources to enable DFU.
+For example, when building from the command line, use the following command:
+
+  .. code-block:: console
+
+     west build -b <BOARD> -p -- -DSB_CONF_FILE="sysbuild-dfu.conf" -DEXTRA_CONF_FILE="overlay-dfu.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-dfu.conf"
+
+.. note::
+   Currently, the nRF53 development kit only supports DFU for the application core.
```

Comment:
nRF5340 or nRF53 Series

Reply:
I think nRF5340 is better here, it's the only supported devkit in this series and will probably always will be the only supported kit in this series.